18/README.md

```diff
@@ -37,27 +37,48 @@ Each Validator checks every new transaction that is about to be committed in a b
 5. If the Election **excluding** the current Vote has less than ⅔ of positive votes and **including** the current Vote has more than ⅔ of positive votes, execute the logic to implement the Election.
 
 ### What is a valid Vote?
-A Validator must be able to discern valid Votes from invalid ones. A valid vote is a transaction that spends one or more Vote Tokens to the Election Address.
+A Validator must be able to discern valid Votes from invalid ones. A valid vote is a transaction where all the following conditions are true.
+
+1. The transaction follows the `TRANSFER` transaction spec.
+1. The transaction additionally follows the [election vote](./election_transaction_vote.yaml) transaction spec.
+1. It spends one or more Vote Tokens to the Election Address.
+
 
 ### What is a valid Election?
 A Validator must be able to discern valid Elections from invalid ones. A valid Election is a transaction where **all** the following conditions are true.
 
-1. `operation` is `CREATE`, or equivalent.
-1. The `inputs` satisfy the conditions for a CREATE transaction. For example, the public key listed in `inputs.owners_before` is the public key of the Initiator.
+1. The transaction complies to the `CREATE` transaction JSON schema.
+1. The transaction additionally complies to the [election JSON schema](#./election_transaction.yaml).
+1. The transaction meets schema requirements, specific to the particular election, if any.
 1. `outputs` has as many entries as the total number of Validators.
 1. Each Validator is represented in `outputs`.
 1. Each entry in `outputs` can be spent by only one Validator, and the amount attached to it is equal to the power of that Validator.
 
-**Note: any change in the Validator Set will make old Elections invalid. Check [approach 2](#generalized-approach-approach-2) for a process that can tolerate a certain degree of change to the Validator Set.**
+**Note: any change in the Validator Set makes old Elections invalid. Check [approach 2](#generalized-approach-approach-2) for a process that can tolerate a certain degree of change to the Validator Set.**
+
+**Note 2: when final votes for multiple elections are present in the same block, all elections that do not change the Validator Set and only one election of those which do change the Validator Set are concluded.**
```

Comment:
I don't understand why we need this note. If there are multiple final votes, they will be executed using the order of the transactions in the block, otherwise we might end up in adding logic to sort elections based on some rules.

Reply:
It explains the extension of this rule which allows elections to be concluded even though one of them changes the validator set. Votes are still processed in their original order but we differentiate between elections that change the validator set and those which do not.

We found it convenient and easy to do but I am also fine with dropping this rule and condemning all the elections that follow the one which changes the validator set to be inconclusive, even within the same block.

Reply:
In my opinion, the simpler we can keep the election protocol the better, for me you can drop this rule.

Reply:
Yeah, I would also prefer the simplicity actually - this way how we handle multiple elections within the same block would be consistent with how we do it across blocks.